tools/lsp/preview/ui.rs

```diff
@@ -563,34 +571,181 @@ fn map_properties_to_ui(
         property_groups.push(property_group_from(&current_group, current_group_properties));
     }
 
-    Some(ElementInformation {
-        id: element.id.clone().into(),
-        type_name: element.type_name.clone().into(),
-        source_uri,
-        source_version,
-        range: to_ui_range(element.range)?,
+    Some((
+        ElementInformation {
+            id: element.id.clone().into(),
+            type_name: element.type_name.clone().into(),
+            source_uri,
+            source_version,
+            range: to_ui_range(element.range)?,
+        },
+        declarations,
+        Rc::new(VecModel::from(property_groups)).into(),
+    ))
+}
 
-        properties: Rc::new(VecModel::from(property_groups)).into(),
-    })
+enum Op {
```

Comment:
this is used only in one function, you could scope it within that function

Reply:
Good point.